instagram/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import PostPages from './components/PostContainer/PostPages.js'
+
+
+class App extends Component {
+	constructor() {
```

Comment:
You can remove this constructor and the contents inside of it. The constructor for a React component is called before it is mounted. Since this is not called before it is mounted, there is no need for it.